### PR TITLE
Fe documentation

### PR DESCRIPTION
**Pull request by stysh98**
UI Documentation for Handling Documents in the Same Spot

### CODE REVIEW

File: server/test/TEstauth.test.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  let sessionCookie: string;
+
+  beforeAll(async () => {
+    // Ensure the database is in a clean state
+    await Database.setup();
+  });
+
+  afterAll(async () => {
+    // Restore the TEST DB to its original state
+    await Database.reset();
```
Comment (emanuelefrisi) [inline server/test/TEstauth.test.ts:19]: Database.reset() is not an existent method

File: server/test/TEstauth.test.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import app from "../src/app";
+import { Database } from "../src/database";
+
+dotenv.config();
+
+describe("Authentication E2E Tests", () => {
+  let sessionCookie: string;
+
+  beforeAll(async () => {
+    // Ensure the database is in a clean state
```
Comment (emanuelefrisi) [inline server/test/TEstauth.test.ts:13]: Can we remove chatgpt-like comments?

File: UI Doc.md
Comment (emanuelefrisi) [inline UI Doc.md:1]: Can you design the figma interface for this UI documentation?

**Review by emanuelefrisi — CHANGES_REQUESTED** [review]
Read the comments

**Review by michepaolo — APPROVED** [review]

File: server/test/TEstauth.test.ts
Comment (emanuelefrisi) [inline server/test/TEstauth.test.ts:1]: Why did you deleted them?

**Review by emanuelefrisi — APPROVED** [review]
Ok

**Pull request merged**